pkg/runtime/runtime.go

```diff
@@ -554,6 +557,13 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 	return nil
 }
 
+// initPluggableComponents discover pluggable components and initialize with their respective registries.
+func (a *DaprRuntime) initPluggableComponents() {
+	if err := pluggable.Discover(context.TODO()); err != nil {
```

Comment:
You should be able to use `a.ctx` here I think. Unless you need a context with a timeout

Reply:
thanks! I was actually trying to find some context to use instead